### ISSUE TITLE
Game crashes regularly with Ticking block entity error

### ISSUE THREAD

**Issue by ddoyle2** [scalda/Foolcraft_3#270]
## Issue Description
Trying to play the new 1.4.5 modpack, the game either freezes or crashes within about 10 minutes

## Reproduce Steps
Log in to my single-player world. Do, anything for 2-15 minutes.  Get a crash or freeze

## Additional Information
https://pastebin.com/7xCYR66z

**Comment by nzminecraftDOTcom** [comment]
Your block fault is at -177,52,19 as shown in the crash report, if you can remove that block using an NBT editor.
This will delete the block from the world.  Make sure you close your game fully before starting this process.

You will want this program here
https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-tools/1262665-nbtexplorer-nbt-editor-for-windows-and-mac

You need to install it.
Locate region file r.-1,0.mca in your world save that you are playing on.
Open that file with the NBTExplorer you just downloaded.
Navigate to the line that ends in "in world at(-12,0)"
Expand that with the +
Expand "Level 12 Entries" (12 may be different on your end)
Expand "Tile entities"
In there you will see "entries" with a number before it. 
Expand them one by one. You will see Coords as the last 3 lines. Match them with the coords in the first line of this guide. 
Delete that entry you expanded showing those coords. 

Then make sure you CTRL+S or File --> Save

Restart Minecraft. Try load your world.

**Comment by scalda** [comment]
Hi can you please test this issue with the new beta version 1.5 as this problem is fixed in both ender io and mob grinding utils and please report back

**Comment by ddoyle2** [comment]
I got the block out of there and haven't had a problem since.  

I'll try updating it within the next couple of days.

**Comment by scalda** [comment]
ok thanks for letting me know

**Comment by ddoyle2** [comment]
Sorry, I think there was a misunderstanding there.  I had not yet updated this when you closed it. The problem is re-occurring.

**Comment by ddoyle2** [comment]
The ticking block entity occurred again after playing for a couple of hours.   I had assumed I knew what was causing it, but I'm not sure I do.  Previously the issue was caused by a mob masher (I think).  I had confirmed it didn't happen when the mob masher was off (several hours).  It also seemed to work when it was on (2 hours).  But I then made a change to a filter that was fairly indirectly attached to the mob masher (through a vacuum hopper and was a filter in a trash can) and things blew up within 20 minutes.  Coincidence?  Possibly!

The issue now occurs within about a minute within logging in.  I get a little time to explore, but not much.  But, I have no idea where the issue is now occurring.  (let me know if you want me to attach new crash reports)

I'll keep trying to get information.  Currently, I've broken the mob masher, and also the resturbed spawner that was feeding it.  If I try to open the diamond chest fed by the. indirect. masher path, the game crashes.  If I try to look at the filter in the trash can, it crashes.  If I try to open my inventory (to verify that this IS a mob masher), the game freezes.  I'm not sure if it is timing, location, or the actual actions, but I've run out of time for tonight.  I'll try again tomorrow.
